src/vars/lists/assoc_lists.md

```diff
@@ -0,0 +1,39 @@
+# Associated lists
+
+## ⚠⚠ WIP PAGE ⚠⚠
+
+Associated lists, or list associations, add a unique functionality to regular lists by allowing you to associate values within the list with another value often referred to in other languages as a `key-value pair`. This can be done as such:
```

Comment:
```suggestion
Associated lists, or list associations, add a unique functionality to regular lists by allowing you to associate values within the list with another value often referred to in other languages as a `key-value pair` or dictionary. This can be done as such:
```